core/lib/multivm/src/versions/vm_fast/tests/l1_tx_execution.rs

```diff
@@ -117,9 +117,8 @@ fn test_l1_tx_execution() {
     let res = vm.vm.execute(VmExecutionMode::OneTx);
     let storage_logs = res.logs.storage_logs;
     let res = StorageWritesDeduplicator::apply_on_empty_state(&storage_logs);
-    // We changed one slot inside contract. However, the rewrite of the `basePubdataSpent` didn't happen, since it was the same
-    // as the start of the previous tx. Thus we have `+1` slot for the changed counter and `-1` slot for base pubdata spent
```

Comment:
Dumb question: Why has the amount of `initial_storage_writes` changed? What is this change caused by?

Reply:
These changes from this [PR](https://github.com/matter-labs/zksync-era/pull/2737) which includes protocol defense updates. We needed them for the CI to pass.